atlasdb-config/src/main/java/com/palantir/atlasdb/factory/DefaultPhysicalPersistentStorageFactory.java

```diff
@@ -25,27 +25,29 @@
 import org.slf4j.LoggerFactory;
 
 import com.palantir.atlasdb.config.RocksDbPersistentStorageConfig;
-import com.palantir.atlasdb.persistent.api.PhysicalPersistentStore;
-import com.palantir.atlasdb.persistent.rocksdb.RocksDbPhysicalPersistentStore;
+import com.palantir.atlasdb.persistent.api.PersistentStore;
+import com.palantir.atlasdb.persistent.rocksdb.RocksDbPersistentStore;
+
+import okio.ByteString;
 
 /**
- * Constructs a new {@link PhysicalPersistentStore} with new persistent storage connection on each call of
+ * Constructs a new {@link PersistentStore} with new persistent storage connection on each call of
  * {@link DefaultPhysicalPersistentStorageFactory#constructPersistentStore(RocksDbPersistentStorageConfig)}.
  */
 public final class DefaultPhysicalPersistentStorageFactory implements PhysicalPersistentStorageFactory {
```

Comment:
Can we just bite the bullet and call this rocks, either that or do the "generification" properly. The mix of the two is inconsistent and quite confusing. Probably a separate PR however.

Reply:
I would do the visitor pattern in a following PR and leave it as is for now.